src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -201,8 +200,8 @@ class DirectoryIterator::NativeIterator::Pimpl
     DIR* dir;
 };
 
-DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard)
-    : pimpl (new DirectoryIterator::NativeIterator::Pimpl (directory, wildCard))
+DirectoryIterator::NativeIterator::NativeIterator (const File& directory, const String& wildCard_)
```

Comment:
@rec If there are legitimate issues (i.e. bugs) caused by shadowing, please address them in a separate pull request.